configuration/mobile/android.mdx

```diff
@@ -10,7 +10,7 @@ Configure Rownd for Android mobile devices including ([React Native](/sdk-refere
 
 ### 1. Configure in Rownd Platform
 
-1. Select **Sign-in methods** from the left navigation menu.
+1. Select **Sign in methods** from the left navigation menu.
```

Comment:
Technically, there should be a hyphen here. Perhaps that's missing from the platform UI...